lib/serverengine/command_sender.rb

```diff
@@ -0,0 +1,76 @@
+#
+# ServerEngine
+#
+# Copyright (C) 2012-2013 FURUHASHI Sadayuki
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+module ServerEngine
+  module CommandSender
+    # requires send_signal method or @pid
+    module Signal
+      private
+      def _stop(graceful)
+        _send_signal(!ServerEngine.windows? && graceful ? Daemon::Signals::GRACEFUL_STOP : Daemon::Signals::IMMEDIATE_STOP)
```

Comment:
`Daemon::Signals::IMMEDIATE_STOP` is assigned dynamically to use `KILL` when it runs on Windows.
Are there any reason not to do it for `GRACEFUL_STOP`? Is it to show "it's not graceful on Windows" explicitly?